app/src/types/genericTypes.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+"use strict";
+
+import t from "flow-runtime";
+import * as v from "flow-runtime-validators";
+
+export const FiscalNumberType = t.refinement(
```

Comment:
please add a short comment describing each type like we do [here](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/utils/strings.ts#L81), this is especially important for "obscure" types like `IssuerType`.

For types defined in other projects, if possible, reference the related spec/documentation.